eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -67,21 +65,18 @@ jobs:
             - RedHat.7.Amd64.Open
             - Ubuntu.1604.Amd64.Open
             - Ubuntu.1804.Amd64.Open
-            - SLES.12.Amd64.Open
             - SLES.15.Amd64.Open
```

Comment:
Should `isFullMatrix==true` be a strict super set of `isFullMatrix==false`?

Reply:
Yes, I've asked that in other PRs. That way we can track pass rate.

Reply:
@jkotas This is game changer. If I understand correctly, we are not able run test only for PR as Rolling tests will run all PR tests, is that correct?

Reply:
Yes, I think it makes sense for PRs to run subset and for rolling to run everything (including the subset that PRs run).

I believe that it is the case today for the most part, there are just a few outliers.

Reply:
@ViktorHofer, If I the a  Debian 10 © as an example: We want this update: `Drop rolling (keep PR)`, was it meant to remove it only from isFullMatrix==true and leave it in isFullMatrix==false ?

Reply:
@aik-jahoda based on the discussion above, we don't want to drop rolling anywhere except for EOL distros. So "Drop rolling (keep PR)" should probably be ignored.